library/src/main/java/com/google/android/exoplayer/text/eia608/ClosedCaptionCtrl.java

```diff
@@ -69,6 +71,45 @@
   public static final byte TAB_OFFSET_CHAN_1 = 0x17;
   public static final byte TAB_OFFSET_CHAN_2 = 0x1F;
 
+  private static final String TAG = "Eia608CaptionControl";
+
+  private static final int[] COLOR_VALUES = {
+          Color.WHITE,
+          Color.GREEN,
+          Color.BLUE,
+          Color.CYAN,
+          Color.RED,
+          Color.YELLOW,
+          Color.MAGENTA,
+          Color.TRANSPARENT // Last color setting should be kept
+  };
+
+  // for debug purposes
```

Comment:
It would be preferable to remove it until it's needed, yes. It makes it easier to find "the change that added positioning" if its contained within a single change, as opposed to some of it arriving first :).